docs/faq/lukso/general-information.md

```diff
@@ -0,0 +1,42 @@
+
+LUKSO also wants to ensure **smooth user onboarding** and subsidize early adopters using Universal Profiles. Creating a network effect of convenience through subsidization would not have been sufficient on existing Layer-1 blockchains due to high gas costs or Universal Profiles being a smaller subset of deployed accounts.
+
+Despite L2 solutions offering lower costs, they often suffer from interoperability issues, governance concerns, and the need for rolling back data to the main network, **risking the siloing of assets and profiles**. Depending on the rollup type, different sacrifices like calculation, smart contract execution, or finality time have to be made on top.
+
+Rollups, while providing scalability, present their challenges. They are only as secure as the speed at which assets can be exited to the main chain. Moreover, **not every protocol or activity** implemented on the rollup **can be exited**, as not everything is an asset. This limits their applicability and functionality for new account systems.
+
+While standalone applications might not see the necessity of having their own Layer-1 network, LUKSO provides a **distinct realm for a new** core blockchain **account ecosystem**, necessarily needing the robust and secure foundation to utilize its full potential.
+
+### What's the token distribution of LUKSO?
+
+Please look at our official [Network Migration](https://etherscan.io/address/0x42000421dd80D1e90E56E87e6eE18D7770b9F8cC#code) article to see the distribution amounts and listing of the final supply. Previous distributions have been outlaid in our [Genesis Validator](https://medium.com/lukso/its-happening-the-genesis-validators-are-coming-ce5e07935df6) announcement.
+
+### What's the difference between LYXt, LYXe, and LYX?
+
+- **LYXt** is the currency for developers used on our official LUKSO Testnet without monetary value. LYXt can be required for free using the [Testnet Faucet](https://faucet.testnet.lukso.network/). The Testnet is maintained by a set of whitepisted node operators to ensure stability.
+- **LYXe** is the representation of LYX on the Ethereum Blockchain, used to start off the LUKSO Mainnet in a distributed way. It was created in 2020 during the [rICO](https://medium.com/lukso/re-launching-the-reversible-ico-5289989ce7ed) of LUKSO and started as main token before the mainnet was released.
```

Comment:
Maybe you can put a link to:

https://etherscan.io/token/0xA8b919680258d369114910511cc87595aec0be6D

Reply:
Maybe you can also word it as _"it's the LUKSO token on Ethereum"_